notebooks/text_classification/text_classification_introspective_rationale_explainer.ipynb

```diff
@@ -137,9 +140,17 @@
    "outputs": [],
    "source": [
     "# get all unique labels\n",
-    "y_labels = all_data[LABEL_COL].unique()\n",
-    "args.labels = np.array(sorted(y_labels))\n",
-    "args.num_labels = len(y_labels)"
+    "y_labels = all_data[LABEL_COL].unique()"
```

Comment:
@janhavi13  Should all these cell output metadata be cleared before merging?

Reply:
@minthigpen - things in source[] is the cell content